<?xml version="1.0"?>
<!--
    This file is part of ALPHA AUV project.

    Author: Ansel Austin
    Email: ansel.austin@uri.edu
    Year: 2024

    Copyright (C) 2022 Smart Ocean Systems Laboratory
-->
<launch>

    <arg name="parent_name" default="wamv"/>
    <arg name="child_name" default="alpha_rise"/>

    <node ns="$(arg parent_name)" pkg="mvp_localization_utilities" type="world_to_world_transform_node" name="world_to_world_tf" output="screen">
        <param name = "child_name" value="$(arg child_name)"/>
        <param name = "parent_name" value="$(arg parent_name)"/>
    </node>

</launch>